### PR TITLE
*: use co as prefix rather than cos

### PR DESCRIPTION
**Pull request by zhexuany**
In this PR, we adopt "co" as the common prefix in our project rather than "cos"

### CODE REVIEW

**Comment by zhexuany** [comment]
@YunxianHua PTAL

File: cobridge_base/src/version.cpp.in [hunk 1]
```diff
@@ -14,12 +14,12 @@
...
 //////////////////////////////////////////////////////////////////////////////////////
 
-#include <cos_bridge.hpp>
+#include <cobridge.hpp>
 
-namespace cos_bridge_base{
+namespace cobridge_base{
 
-        const char COS_BRIDGE_VERSION[] = "@COS_BRIDGE_VERSION@";
+        const char cobridge_VERSION[] = "@cobridge_VERSION@";
```
Comment (YunxianHua) [inline cobridge_base/src/version.cpp.in:21]: COBRIDGE_VERSION

File: package.xml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 <package format="3">
-    <name>cos-bridge</name>
+    <name>cobridge</name>
     <version>0.0.1</version>
     <description>bridge for ROS2</description>
```
Comment (YunxianHua) [inline package.xml:5]: 这里帮我改一下吧，现在已经不是 for ROS2 了

File: ros1_bridge/src/ros1_bridge_nodelet.cpp [hunk 3]
```diff
@@ -113,15 +113,15 @@ class CosBridge : public nodelet::Nodelet {
     }
 
     const char* rosDistro = std::getenv("ROS_DISTRO");
-    ROS_INFO("Starting cos_bridge (%s, %s@%s) with %s", rosDistro,
-             cos_bridge_base::COS_BRIDGE_VERSION, cos_bridge_base::COS_BRIDGE_GIT_HASH,
-             cos_bridge_base::websocket_user_agent());
+    ROS_INFO("Starting cobridge (%s, %s@%s) with %s", rosDistro,
+             cobridge_base::cobridge_VERSION, cobridge_base::cobridge_GIT_HASH,
+             cobridge_base::websocket_user_agent());
```
Comment (YunxianHua) [inline ros1_bridge/src/ros1_bridge_nodelet.cpp:118]: COBRIDGE_VERSION  and COBRIDGE_GIT_HASH

**Comment by zhexuany** [comment]
This PR can compiled with ROS2 humble not yet verify with foxy. I do not have foxy env, maybe you can verify this in your env @YunxianHua

File: cobridge_base/src/version.cpp.in (hunk 1 above)
Comment (zhexuany) [reply cobridge_base/src/version.cpp.in]: already resolved.

File: ros1_bridge/src/ros1_bridge_nodelet.cpp (hunk 3 above)
Comment (zhexuany) [reply ros1_bridge/src/ros1_bridge_nodelet.cpp]: sure. Already changed.

**Pull request merged**